src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,28 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { getUserById } from '../../services/userService';
+import { Todo } from '../../types/todo';
+import { User } from '../../types/user';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const user: User | null = getUserById(todo.userId);
```

Comment:
Avoid calling `getUserById` directly in this component. Instead, pass the user data as a prop from the parent component to adhere to the checklist requirement of centralizing data preparation.